src/arch/aarch64/vgic/vdist.h

```diff
@@ -164,6 +164,11 @@ static void vgic_dist_disable_irq(vgic_t *vgic, size_t vcpu_id, int irq)
      * part of the platform initialization, no dedicated messages are logged
      * here to avoid bloating the logs.
      */
+    // @ericc: Hack to not let UIO disable their IRQ, should be done by registering
```

Comment:
This cannot be merged. We can let the bug through as the UIO drivers are still somewhat functional and instead make a GitHub issue about the problem with UIO IRQs.